packages/kbn-check-mappings-update-cli/current_mappings.json

```diff
@@ -993,6 +993,52 @@
     "dynamic": false,
     "properties": {}
   },
+  "entity-definition": {
```

Comment:
are these new in this PR (as opposed to renamed)? what are they used for?

Reply:
it was added in this PR which is surprising given the saved object existed before, and tbh I don't know how it was added.
 @elastic/kibana-core do you know what this is used for ?

Reply:
> which is surprising given the saved object existed before

Was the SO registration previously under some condition, like some flag or config check that could have been disabled with the default config? Our SO scanner is basically just starting Kibana with the default config and all plugins forcefully enabled.

> do you know what this is used for ?

Yeah, we're generating those files as references for some SO related checks (mapping compatibility checks and model version inclusion checks)

Reply:
> Was the SO registration previously under some condition, like some flag or config check that could have been disabled with the default config

Yes the plugin was enabled by a setting and disabled by default before this change. Thanks @pgayvallet